packs/data/bestiary-effects.db/effect-ectoplasmic-form-physical.json

```diff
@@ -0,0 +1,76 @@
+{
+    "_id": "QMcmhifNQOPguv2t",
+    "img": "systems/pf2e/icons/spells/dinosaur-form.webp",
+    "name": "Effect: Ectoplasmic Form (Physical)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>The creature looses the incorporeal trait, the immunity to precision damage and all resistances. It gains temporary hit points and an increased armor class arroding to the specific creature, its jaws strikes deal piercing and its claws strikes deal slashing damage instead of negative.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "ActorTraits",
+                "remove": [
+                    "incorporeal"
+                ]
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.attributes.ac.value",
+                "value": "@actor.system.custom.modifiers.ectoplasmicFormAC"
+            },
+            {
+                "key": "TempHP",
+                "value": "@actor.system.custom.modifiers.ectoplasmicFormTempHP"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "ectoplasmic-form",
+                "value": true
+            },
```

Comment:
```suggestion
```
Effects set their own roll options automatically

Reply:
Yeah - I forgot the effect creates its own flag I can predicate against :-) I'll change that as a whole.